<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Move Recorder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Motor Group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="motor_group_list">
         <item>
          <property name="text">
           <string>Head</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Torso</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Arm</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Arm</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Leg</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Leg</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="compliant_box">
         <property name="text">
          <string>Compliant</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Filename:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filename_field">
           <property name="text">
            <string>my_move</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="record_button">
          <property name="font">
           <font>
            <pointsize>36</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Record</string>
          </property>
         </widget>
         <widget class="QPushButton" name="stop_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>36</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Recorded Move</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="move_list">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="loop_box">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Loop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="play_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_button_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
